src/components/Message.jsx

```diff
@@ -0,0 +1,20 @@
+import React, {PureComponent} from "react"
```

Comment:
Its a minor, but I can suggest to change this on just import React from "react"and useextends React.PureComponentfor uniformity of code, and makeimport React` usefull =)

Reply:
additional point
1. React is always required when you use jsx because jsx is transpile to createElement https://babeljs.io/repl#?browsers=&build=&builtIns=false&spec=false&loose=false&code_lz=JYWwDg9gTgLgBAJQKYEMDG8BmUIjgcilQ3wG4AocgCgEo4BeAPjgB4ATYAN0YBUkBnGCwD0HbkA&debug=false&forceAllTransforms=false&shippedProposals=false&circleciRepo=&evaluate=false&fileSize=false&timeTravel=false&sourceType=module&lineWrap=true&presets=es2015%2Creact%2Cstage-2&prettier=false&targets=&version=7.10.5&externalPlugins=
2. in file above you don't import Component as named import

Reply:
Thanks for your notes. I got it and fixed it.